includes/classes/Coupon.php

```diff
@@ -202,7 +202,7 @@ public static function generateRandomCouponCode(string $salt = "secret", $length
         $good_result = 0;
         $new_code = '';
         while ($good_result === 0) {
-            $random_start = @rand(0, (128 - $length));
+            $random_start = @random_int(0, (128 - $length));
```

Comment:
Why continuing with the error-suppression?

Reply:
Removed.